packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -0,0 +1,655 @@
+                source='track page'
+              />
+            ))}
+        </div>
+      )
+    )
+  }
+
+  const renderReleased = () => {
+    return (
+      !isUnlisted &&
+      released && (
+        <InfoLabel
+          className={styles.infoLabelPlacement}
+          labelName='released'
+          labelValue={formatDate(released)}
+        />
+      )
+    )
+  }
+
+  const renderStatsRow = () => {
+    const isLongFormContent =
+      genre === Genre.PODCASTS || genre === Genre.AUDIOBOOKS
+    const isNewPodcastControlsEnabled = getFeatureEnabled(
+      FeatureFlags.PODCAST_CONTROL_UPDATES_ENABLED,
+      FeatureFlags.PODCAST_CONTROL_UPDATES_ENABLED_FALLBACK
+    )
+
+    return (
+      <>
+        <RepostFavoritesStats
+          isUnlisted={isUnlisted}
+          repostCount={repostCount}
+          saveCount={saveCount}
+          onClickReposts={onClickReposts}
+          onClickFavorites={onClickFavorites}
+        />
+        {isLongFormContent && isNewPodcastControlsEnabled
+          ? renderListenCount()
+          : null}
+      </>
+    )
+  }
+
+  const renderDownloadButtons = () => {
+    return (
+      <DownloadButtons
+        className={styles.downloadButtonsContainer}
+        trackId={trackId}
+        isOwner={isOwner}
+        following={following}
+        doesUserHaveAccess={doesUserHaveAccess}
+        onDownload={onDownload}
+      />
+    )
+  }
+
+  const isLoading = loading || artworkLoading
+  const dogEarType = isLoading
+    ? undefined
+    : getDogEarType({
+        premiumConditions,
+        isUnlisted,
+        isOwner,
+        doesUserHaveAccess
+      })
+
+  const overflowMenuExtraItems = []
+  if (!isOwner) {
+    overflowMenuExtraItems.push({
+      text: following ? 'Unfollow Artist' : 'Follow Artist',
+      onClick: () =>
+        setTimeout(() => (following ? onUnfollow() : onFollow()), 0)
+    })
+  }
+
+  const overflowMenu = {
+    menu: {
+      type: 'track',
+      trackId,
+      trackTitle,
+      genre,
+      handle: artistHandle,
+      isFavorited: isSaved,
+      mount: 'page',
+      isOwner,
+      includeFavorite: false,
+      includeTrackPage: false,
+      isArtistPick,
+      includeEmbed: !(isUnlisted || isPremium),
+      includeArtistPick: !isUnlisted,
+      includeAddToPlaylist: !(isUnlisted || isPremium),
+      extraMenuItems: overflowMenuExtraItems
+    }
+  }
+
+  const fadeIn = {
+    [styles.show]: !isLoading,
+    [styles.hide]: isLoading
+  }
+
+  return (
+    <Tile
+      className={styles.giantTrackTile}
+      dogEar={dogEarType}
+      size='large'
+      elevation='mid'
+    >
+      <div className={styles.topSection}>
+        <GiantArtwork
+          trackId={trackId}
+          coverArtSizes={coverArtSizes}
+          coSign={coSign}
+          callback={onArtworkLoad}
+        />
+        <div className={styles.infoSection}>
+          <div className={styles.infoSectionHeader}>
+            {renderCardTitle(cn(fadeIn))}
```

Comment:
Had to add a `cn()` here because the helper function expects a simple string classname and `fadeIn` is an object with conditional classnames meant to be passed to the `classnames` function.